crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -21,37 +23,17 @@ use tracing::{
 };
 
 use crate::{
-    base64_string::Base64String,
     data_availability::CelestiaClient,
-    keys::{
-        private_key_bytes_to_keypair,
-        validator_hex_to_address,
-    },
     sequencer::SequencerClient,
     sequencer_block::SequencerBlock,
+    validator,
```

Comment:
is there a reason this can't be `validator::Validator` shortening up the references below?